.github/workflows/build-ci-gcc-centos7.yaml

```diff
@@ -0,0 +1,73 @@
+name: Build GCC on centos7
+run-name: Build GCC on centos7
+on:
+  push:
+    paths:
+      - .github/services/Dockerfile.*
+    branches:
+      - master
+  pull_request:
+
+env:
+  # REGISTRY: ghcr.io
+  IMAGE_NAME: gzigzigzeo/gcc-centos7
+  GCC_VERSION: 4.9.4
```

Comment:
nit: Why are we using GCC 4.9? If we're building from the source, we can use 12.2.